Add sv_SE locale

vladdeSV opened the pull request:
This PR contains most translations to Swedish (sv_SE).

---

Most keys have a translation, but a few are missing. I am unsure what the meaning behind the original keys are, so I've left them out.

There are also two words which I do not know how to translate (I do not believe there are Swedish equivalents):
- gutter
- nested

Because many of the words do not have a direct translation to Swedish I've done my best to translate them, but I highly suggest some other Swedish persons give their opinions as well. The following "odd" translations are used (in order from top to bottom of the most strange translation IMO):
- locale → översättningsväg
- locale file → översättningsvägsfil
- source language → källspråk
- namespace → utrymme
- star on github → stjärna på github
- key → nyckel
- review → granskning
- resolve → fixa
- request → efterfrågan

vladdeSV commented:
(Sidenote: I interpreted the key `prompt.choice_key_to_insert` as "Choose key to insert". Please correct me if this is wrong.)

antfu commented on locales/sv-se.json:
```diff
@@ -0,0 +1,198 @@
+  "button.toastMessage": "",
+  "clicked.times-0": "",
+  "clicked.times-1": "",
+  "clicked.times-2": "",
```
 😜

vladdeSV replied:
Sorry, I didn't understand you.

I copied those from `en.json`, and they did not have a translation there. Should they have a translation?

antfu commented:
> namespace → utrymme

Maybe we don't need to translate this? I think it's more like a term than an actually meaningful word.

vladdeSV commented:
I changed "namnutrymme" to "\`namespace\`" (with backticks) to indicate it's a term. I did not mention this previously, but I also changed "glob" to "\`glob\`".

Let me know what you think about this.

vladdeSV commented:
I believe this is good enough to be merged, and the only *real* oddity is the translation of "locale" (to "översättningsväg"). However, I am sure that if someone has a better translation they'll fix with a PR.

I am okay with this being merged 😄

antfu commented:
Great! Thanks a lot!
